### DISCUSSION TITLE
What is Build_type?

### DISCUSSION THREAD

**Discussion by ceanwang** [OpenRadioss/OpenRadioss#423] [Q&A] [answered]
Hi,
In CMakeLists.txt:
`if ( NOT DEFINED COM )

   message (STATUS "Build_type : o")

else()

   message (STATUS "Build_type : c")

endif()`

What this Build_type (COM) mean? 
Cheers,
Cean

**Comment by elequiniou** [comment, marked as answer]
"COM" is only defined for the commercial version ("Build_type : c"), for the open-source version("Build_type : o"), it is not defined, so please ignore. Thanks!

**Reply by ceanwang** [reply]
Thanks. That answered my question what com means.
